docs/csharp/language-reference/keywords/private-protected.md

```diff
@@ -60,3 +70,4 @@ The `private protected` keyword combination is a member access modifier. A priva
  [public](../../../csharp/language-reference/keywords/public.md)   
  [private](../../../csharp/language-reference/keywords/private.md)   
  [internal](../../../csharp/language-reference/keywords/internal.md)
+ [Security concerns for internal virtual keywords](https://msdn.microsoft.com/library/heyd8kky(v=vs.110))
```

Comment:
The added link is on the same line with the line above. See the internal URL I added.